### PR TITLE
Fix nan assignment to integer array in RaggedArray

### PR DESCRIPTION
**Pull request by philippjfr**
Fixes tests

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -45,13 +43,13 @@
...
         'fastparquet >=0.1.6',  # optional dependency
         'holoviews >=1.10.0',
+        'bokeh',
         'pyarrow',
         'netcdf4',
         'twine',   # required for pip packaging
         'rfc3986', # required by twine
         'keyring', # required by twine
-        'spatialpandas',
-        'ipympl'
```
Comment (jbednar) [inline setup.py:54]: Why was this removed? Isn't it needed for the datashader examples now?

Comment (philippjfr) [reply setup.py:54]: ipympl is not enabled by default no and I was trying to improve solve time.
